src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="content page__content">
+    <article class="calendar content__calendar calendar--start-day--mon">
```

Comment:
The month-length modifier is missing from the calendar. Add the modifier with values 28, 29, 30, and 31 using @for.